guava/src/com/google/common/collect/ImmutableList.java

```diff
@@ -166,7 +167,7 @@
    *
    * @throws NullPointerException if any element is null
    */
-  public static <E> ImmutableList<E> of(E e1, E e2, E e3, E e4, E e5, E e6, E e7, E e8, E e9) {
+  public static <E extends @NonNull Object> ImmutableList<E> of(E e1, E e2, E e3, E e4, E e5, E e6, E e7, E e8, E e9) {
```

Comment:
The Google Java style requires lines of code to break before 100 characters.  See how the next `of` method was formatted, and do so for the lines that are longer than 100 characters.

(But, in this case `extends @NonNull Object` seems unnecessary.)